src/picotorrent/ui/mainframe.cpp

```diff
@@ -444,7 +447,14 @@ void MainFrame::AddTorrents(std::vector<lt::add_torrent_params>& params)
         auto our = new BitTorrent::AddParams();
 
         p.flags |= lt::torrent_flags::duplicate_is_error;
-        p.save_path = m_cfg->Get<std::string>("default_save_path").value();
+        if (!m_options.save_path.empty())
```

Comment:
What would happen here if you start the first instance with `--save_path C:\foo`? I think it would always use the provided save path?

Reply:
Yeah you're right , have to fix that